src/dev/i18n/utils.js

```diff
@@ -27,15 +27,27 @@ import {
 import fs from 'fs';
 import glob from 'glob';
 import { promisify } from 'util';
+import normalize from 'normalize-path';
+import path from 'path';
 
 const ESCAPE_LINE_BREAK_REGEX = /(?<!\\)\\\n/g;
 const HTML_LINE_BREAK_REGEX = /[\s]*\n[\s]*/g;
 const LINE_BREAK_REGEX = /\n/g;
 
 export const readFileAsync = promisify(fs.readFile);
 export const writeFileAsync = promisify(fs.writeFile);
+export const makeDirAsync = promisify(fs.mkdir);
+export const accessAsync = promisify(fs.access);
 export const globAsync = promisify(glob);
 
+export function normalizePath(inputPath) {
```

Comment:
__nit:__ please add tests for all new _exported_ functions (except for the ones re-exported with promisify obviously).